Parser/spark.py

```diff
@@ -30,7 +32,7 @@ def _namelist(instance):
         for b in c.__bases__:
             classlist.append(b)
         for name in c.__dict__.keys():
```

Comment:
`.keys()` is redundant.

Reply:
Yep. But this is not a rewrite of the `Parser/spark.py` module. I did not touch this line.